### PR TITLE
fix(shell-center-row): Error when no action-bar is provided

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #1031

## Summary

fix(shell-center-row): Error when no action-bar is provided

### CODE REVIEW

**Comment by driskull** [comment]
@jcfranco can you review the test? I'm not sure if this is the best way to detect the error but I could not find another solution. 

If we do want to go with this, it seems like a good candidate to be something we test for automatically for each component.

File: src/components/calcite-shell-center-row/calcite-shell-center-row.e2e.ts [hunk 1]
```diff
@@ -24,6 +24,19 @@ describe("calcite-shell-center-row", () => {
       }
     ]));
 
+  it("should not error when there is no action-bar", async () => {
```
Comment (driskull) [inline src/components/calcite-shell-center-row/calcite-shell-center-row.e2e.ts:27]: @jcfranco could we make this test reusable like the `renders` test? I don't think any components should throw a console.error.

Comment (jcfranco) [reply src/components/calcite-shell-center-row/calcite-shell-center-row.e2e.ts:27]: Looks like there's already a way to fail on console error:

```ts
it("should not error when there is no action-bar", async () =>
  await newE2EPage({
    html: "<calcite-shell-center-row></calcite-shell-center-row>",
    failOnConsoleError: true
  }));
```

Comment (driskull) [reply src/components/calcite-shell-center-row/calcite-shell-center-row.e2e.ts:27]: Oh nice. Ill test it out. Can we turn it on by default?

Comment (jcfranco) [reply src/components/calcite-shell-center-row/calcite-shell-center-row.e2e.ts:27]: Hmm... I don't see a way to enable it globally.

Comment (driskull) [reply src/components/calcite-shell-center-row/calcite-shell-center-row.e2e.ts:27]: Can we maybe add this to the renders test?

**Pull request merged**